index.html

```diff
@@ -24,24 +47,26 @@ <h2>The Past</h2>
                 <p>Proin sed quam sed tellus vestibulum ultrices quis in nunc. Phasellus id dui id tortor tincidunt efficitur. Proin faucibus imperdiet erat, non varius lacus. Maecenas non nisl id turpis egestas tincidunt. Nam condimentum venenatis magna eget finibus.</p>
             </div>
         </section>
-        
+
         <section class="middle-content">
-        
+
             <h2>Why Did It Have To Be Boxes...</h2>
-            
+
+            <!--I think it will be simpler to directly style the boxes here so that's what I'm gonna do.-->
+
             <div class="boxes">
-                <div class="box">Box 1</div>
-                <div class="box">Box 2</div>
-                <div class="box">Box 3</div>
-                <div class="box">Box 4</div>
-                <div class="box">Box 5</div>
-                <div class="box">Box 6</div>
-                <div class="box">Box 7</div>
-                <div class="box">Box 8</div>
-                <div class="box">Box 9</div>
-                <div class="box">Box 10</div>
+                <div class="box" style="background-color:teal;";>Box 1</div>
+                <div class="box" style="background-color:gold;">Box 2</div>
+                <div class="box" style="background-color:cadetblue;">Box 3</div>
+                <div class="box" style="background-color:coral;">Box 4</div>
+                <div class="box" style="background-color:crimson;">Box 5</div>
+                <div class="box" style="background-color:forestgreen;">Box 6</div>
+                <div class="box" style="background-color:darkorchid;">Box 7</div>
+                <div class="box" style="background-color:hotpink;">Box 8</div>
+                <div class="box" style="background-color:indigo;">Box 9</div>
+                <div class="box" style="background-color:dodgerblue;">Box 10</div>
             </div>
```

Comment:
while this works, inline-styles are the heaviest and should be avoided if possible. you could add in classes onto the boxes and put the colors in the css file